docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -36,122 +40,131 @@ Available devices:
-- FakeQuantize
 
-> **NOTE**: You can have any number and order of simple layers.
+## Supported features
 
-A combination of a Convolution layer and simple layers results in a single fused layer called
-*Convolution*:
-![conv_simple_01]
+### Heterogeneous execution
+GPU plugin supports `ov::Core::query_model()` method, thus it can be used for heterogeneous execution via specifying
+`"HETERO:GPU,CPU"` device instead of `"GPU"`. In that case layers that are not implemented in GPU plugin will be executed via CPU plugin.
 
+See [Heterogeneous execution page](../hetero_execution.md) for more details.
 
-### Fusing Pooling and FakeQuantize Layers
+### Multi-device execution
+If a machine has multiple GPUs (for example integrated GPU and discrete Intel GPU), then any supported model can be executed on all GPUs simultaneously.
+This can be achieved by specifying `"MULTI:GPU.1,GPU.0"` as a target device`
 
-A combination of Pooling and FakeQuantize layers results in a single fused layer called *Pooling*:
-![pooling_fakequant_01]
+See [Multi-device execution page](../multi_device.md) for more details.
 
-### Fusing Activation Layers
+### Automatic batching
+GPU plugin is capable of reporting `ov::max_batch_size` and `ov::optimal_batch_size` metrics with respect to the current hardware platform and model,
+thus automatic batching can be applied in cases when `ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)` is set
+or device is specified as `"BATCH:GPU"`.
 
-Given the linear pattern, an Activation layer can be fused into other layers:
+See [Automatic batching page](../automatic_batching.md) for more details.
 
-![fullyconnected_activation_01]
+### Multi-stream execution
+If either `ov::num_streams(n_streams)` with `n_streams > 1` or `ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT)` property is set for GPU plugin,
+then multiple streams are created for the model. In case of GPU plugin each stream has its own host thread and associated OpenCL queue
+which means that incoming infer requests can be processed simultaneously.
 
+> **NOTE**: Simultaneous scheduling of kernels to different queues doesn't mean that the kernels are actually executed in parallel on GPU device. The actual behvaior depends on the hardware architecture, and in some cases the execution may be serialized inside the GPU driver.
 
-### Fusing Convolution and Sum Layers
+Implementation of streams in GPU plugin supports constant memory sharing across streams.
 
-A combination of Convolution, Simple, and Eltwise layers with the sum operation results in a single layer called  *Convolution*:
-![conv_sum_relu_01]
+See [optimization guide](@ref openvino_docs_deployment_optimization_guide_dldt_optimization_guide) for more details.
 
-### Fusing a Group of Convolutions
+### Dynamic shapes
+GPU plugin supports dynamic batch only with fixed upper bound. Any other dynamic dimensions are unsupported. For dynamic batch case GPU plugin creates
```

Comment:
Please be more specific about this `batch`. As far as I know this is not always 0-th dimension. And we don't need to mark this dimension as `batch` in layout. So term `batch` here is not well specified.

Reply:
Please check updated version. I've mentioned that batch is specified as `N` in ov layout and added reference to layout overview page.